wrappers/binaries.go

```diff
@@ -59,7 +60,7 @@ func AddSnapBinaries(s *snap.Info) (err error) {
 		}
 		created = append(created, wrapperPath)
 
-		if noCompletion || app.Completer == "" {
+		if !release.OnClassic || noCompletion || app.Completer == "" {
```

Comment:
I think a comment is appropriate here. Even someone following git blame will find little context as to why this is required.